backend/src/main/java/hu/flowacademy/meteo/dto/BetweenDatesDto.java

```diff
@@ -0,0 +1,21 @@
+package hu.flowacademy.meteo.dto;
+
+import hu.flowacademy.meteo.model.enumPackage.Type;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Date;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class BetweenDatesDto {
+
+    Long id;
```

Comment:
Ez az ID itt mit akar repreyentalni minek az id-ja ez?

Reply:
a station-nek, átírom majd stationId-ra.

Reply:
Szerintem a station ID nem a between dates része, mindjárt írok kommentet még a controllerhez.

Reply:
requestFilter object-kent szerintem maradhat.

Reply:
Átnevezve (duplán  :) ) szerintem is maradhat.